sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -508,8 +509,10 @@ def create_item(
         if indexing_directive is not None:
             request_options["indexingDirective"] = indexing_directive
 
+        passthrough_args = {k: v for k, v in kwargs.items() if k != "enable_automatic_id_generation"}
```

Comment:
This like shouldn't be necessary as `enable_automatic_id_generation` has already been `popped` from the `kwargs` above.

Reply:
ah I think I saw this error before I understood what the difference between the `whl` and `sdist` tests were. I never rebuilt so I kept getting the error :) thanks!

Reply:
fixed